buildenv/Build_Instructions_V11.md

```diff
@@ -0,0 +1,457 @@
+
+Two Java builds are produced: a full developer kit (jdk) and a runtime environment (jre)
+- **build/aix-ppc64-normal-server-release/images/jdk**
+- **build/aix-ppc64-normal-server-release/images/jre**
+
+### 5. Test
+:blue_book:
+For a simple test, try running the `java -version` command.
+Change to the /jdk directory:
+```
+cd build/aix-ppc64-normal-server-release/images/jdk
+```
+Run:
+```
+./bin/java -version
+```
+
+Here is some sample output:
+
+```
+openjdk version "10-internal" 2018-03-20
+OpenJDK Runtime Environment (build 10-internal+0-adhoc..openj9-openjdk-jdk10)
+Eclipse OpenJ9 VM (build master-4ecf784, JRE 10 AIX ppc64-64 Compressed References 20180411_000000 (JIT enabled, AOT enabled)
+OpenJ9   - 4ecf784
+OMR      - cb9e24d
+JCL      - eaa06eb based on jdk-10+46)
+```
+:blue_book: *Congratulations!* :tada:
+
+----------------------------------
+
+## Windows
+:ledger:
+
+The following instructions guide you through the process of building a Windows **OpenJDK V11** binary that contains Eclipse OpenJ9. This process can be used to build binaries for Windows 7, 8, and 10.
+
+### 1. Prepare your system
+:ledger:
+You must install a number of software dependencies to create a suitable build environment on your system:
+
+- [Cygwin](https://cygwin.com/install.html), which provides a Unix-style command line interface. Install all packages in the `Devel` category. Included in this package is mingw, a minimalist subset of GNU tools for Microsoft Windows. OpenJ9 uses the mingw/GCC compiler and is tested with version 6.4.0 of same. Other versions are expected to work also. In the `Archive` category, install the packages `zip` and `unzip`. In the `Utils` category, install the `cpio` package. Install any further package dependencies that are identified by the installer. More information about using Cygwin can be found [here](https://cygwin.com/docs.html).
+- [Windows JDK 10](https://adoptopenjdk.net/releases.html?variant=openjdk10#x64_win), which is used as the boot JDK.
+- [Microsoft Visual Studio 2013]( https://go.microsoft.com/fwlink/?LinkId=532495), which is the same compiler level used by OpenJDK; or [Microsoft Visual Studio 2017]( https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=Community&rel=15), which is now supported.
+- [Freemarker V2.3.8](https://sourceforge.net/projects/freemarker/files/freemarker/2.3.8/freemarker-2.3.8.tar.gz/download)
+- [mingw-w64](https://sourceforge.net/projects/mingw-w64/files/Toolchains%20targetting%20Win32/Personal%20Builds/mingw-builds/installer/mingw-w64-install.exe)
+
+**Note**
+1) mingw-w64 must be installed if compiled with VS2017 in that the current version of cygwin has not yet integrated the latest changes of mingw-w64 for the moment. In addition, set up the environment variable MINGW64_BIN_PATH to the binary path of mingw-w64. e.g.
```

Comment:
compiled -> compiling

I assume mingw-w64 contains executables that conflict with cygwin?
MINGW64_BIN_PATH is optional? i.e. if mingw-w64 is before cygwin on the path then setting MINGW64_BIN_PATH is not required?

Reply:
>compiled -> compiling
Updated.

Reply:
> I assume mingw-w64 contains executables that conflict with cygwin? ...
The reason will we choose the latest version of mingw-w64 is we need the new version of x86_64-w64-mingw32-g++ (which is outdated in cygwin) to compile Interpreter related files (including BytecodeInterpreter, MHInterpreter, etc)

There is no need to set up path to Cygwin at all because it knows where to run x86_64-w64-mingw32-g++ in Cygwin, which means there is no way to set up mingw-w64 is before cygwin

Reply:
Already recompiled and it works as expected (also no problem with java -version).
So updated the document and removed MINGW64_BIN_PATH in runtime\makelib\mkconstants.mk.ftl.